soc/arm/st_stm32/stm32h7/Kconfig.defconfig.stm32h730xxq

```diff
@@ -0,0 +1,14 @@
+# ST STM32H730XXQ MCU configuration options
+
+# Copyright (c) 2023 Chen Xingyu <hi@xingrz.me>
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_STM32H730XXQ
```

Comment:
Any comment could help to explain what a new serie is added here which is exactly the same as the existing `STM32H730XX`